vialerbuild

```diff
@@ -28,18 +28,21 @@ vialerbuild info arm64    # show architecture information for arm64 libs.
 vialerbuild info i386     # show architecture information for i386 libs.
 vialerbuild info x86_64   # show architecture information for x86_64 libs.
 
-vialerbuild complete	  # Downloads, builds and copies files.
-vialerbuild rebuild		  # Delete old framework files and rebuild for all architectures
+vialerbuild complete ABSOLUTE_PATH_TO_DEPENDENCY_LIB   # Downloads, builds and copies files with opus lib
+vialerbuild rebuild	ABSOLUTE_PATH_TO_DEPENDENCY_LIB	  # Delete old framework files and rebuild for all architectures with opus lib
```

Comment:
The positioning of the arguments become more important at this moment, and it is required now to add a path to an opus directory, which might not be needed by everyone who is using the script. I think we need a better solution for these kind of options.